sdk/identity/azure-identity/azure/identity/_credentials/default.py

```diff
@@ -70,6 +70,8 @@ class DefaultAzureCredential(ChainedTokenCredential):
         AZURE_TENANT_ID, if any. If unspecified, users will authenticate in their home tenants.
     :keyword str managed_identity_client_id: The client ID of a user-assigned managed identity. Defaults to the value
         of the environment variable AZURE_CLIENT_ID, if any. If not specified, a system-assigned identity will be used.
+    :keyword str interactive_browser_client_id: The client ID to be used in interactive browser credential. If not
+        specified, a system-assigned identity will be used.
```

Comment:
System and user assigned identities apply only to managed identity
```suggestion
        specified, users will authenticate to an Azure development application.
```